modules/gapi/perf/common/gapi_video_perf_tests_inl.hpp

```diff
@@ -162,14 +162,14 @@ PERF_TEST_P_(BackgroundSubtractorPerfTest, TestPerformance)
     initTestDataPath();
 
     gvideo::BackgroundSubtractorType opType;
+    std::string filePath = "";
     bool detectShadows = false;
     double learningRate = -1.;
-    std::string filePath = "";
     std::size_t testNumFrames = 0;
```

Comment:
I think, this variable can be equal to 5. You use one test parameter for this variable.

Reply:
You are right that the only value used for this variable is 5
But I'd like to preserve it as a parameter to be simpler for someone to change the amount of frames under the test cycle. Or to test different number of frames
Anyway, it is not really important where to define its value.